src/mappings/handlers/blockHandlers.ts

```diff
@@ -60,18 +60,18 @@ async function _handleBlock(block: SubstrateBlock): Promise<void> {
         await tranche.computeYieldAnnualized('yield90DaysAnnualized', blockPeriodStart, daysAgo90)
         await tranche.save()
       }
-
+      // Asset operations
       const activeLoanData = await pool.getPortfolio()
```

Comment:
@hieronx is it ok to only consider the active assets from the portfolio call for this?